### ISSUE TITLE
Autoleveling button do nothing

### ISSUE THREAD

**Issue by eclips16** [rhapsodyv/Marlin#10]
I use
#define V6_330_TITAN_TMC 1
#define TFT_LVGL_UI

I am trying to run the autolevevel by using the button in the tools menu bug nothing happen even after configuring the gcode command for autolevel (G28;G29)

**Comment by blayno** [comment]
yes i have this same problem also for LVGL UI bilinear autolevel, i wonder if there will be a fix soon

Expected behavior: Autolevel from LCD

Actual behavior: Nothing happens

**Comment by dandantsui** [comment]
perhaps if @rhapsodyv merged `Marlin/Marlin/src/lcd/extui/lib/mks_ui/` from main Marlin branch -- it would be fixed?

**Comment by kondorzs** [comment]
Hi I have Auto level button problem, but I upgraded more and more and sometimes work it is very interesting. I have problem Cura MKS plugin  (prewiew) no work in file browser just work and Show image on printing screen.
I'm translating now Hungarian (TFT) language and I would like to change Traditional Chinese to Hungarian.
I tested more and look for problem Autolevel and Prewiew.

**Comment by PY2BND** [comment]
Yes, I had the same finding, ABL under TFT_LVGL_UI isn't reacting on the screen.

**Comment by Pigi-102** [comment]
I've the same issue, but have a finding too.
I have serial connected and when I push the ABL button from UI, in the serial output i get:
`ok
echo:Unknown command: "ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ"
ok
`
So it's probably some assignment wrong in the routine

**Comment by m0uh85** [comment]
is the problem fixed? because I prefer the LVGL UI instead the COLOR UI but without ABL makes no sense to compilie with this UI

**Comment by Pedro8005x** [comment]
Hi,
When I selected AUTO_BED_LEVELIG_BILINEAR, there is manual bed level icons
for corners under abl button, but mesh is unavailable for me if LVGL is
selected. If thete is any news on this topic, I'm interested too...

m0uh85 ***@***.***> ezt írta (időpont: 2021. ápr. 23., Pén
12:59):

**Comment by m0uh85** [comment]
I compiled last week with LVGL and used the ABL many times without any problems - seems to be fixed!

**Comment by rossosat** [comment]
> 

Can u help me? i have the same problem

**Comment by Pedro8005x** [comment]
Hi. Try to make an auto home and then push auto bed level button. No any
GUI response, but worked for me. All in all, a flat bed is the best with
manual adjustments, i switched off abl completely...

rossosat ***@***.***> ezt írta (időpont: 2022. febr. 1., K
20:04):
